src/NET8MinimalAPISnapSnart/Shared/DataAccess/DynamoDbProducts.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Amazon.DynamoDBv2;
+using Amazon.DynamoDBv2.Model;
+using Microsoft.Extensions.Logging;
+using Shared.Models;
+
+namespace Shared.DataAccess
+{
+    public class DynamoDbProducts : ProductsDAO
+    {
+        private static string PRODUCT_TABLE_NAME = Environment.GetEnvironmentVariable("PRODUCT_TABLE_NAME");
+        private readonly AmazonDynamoDBClient _dynamoDbClient;
+        private readonly ILogger<DynamoDbProducts> _logger;
+
+        public DynamoDbProducts(AmazonDynamoDBClient dynamoDbClient, ILogger<DynamoDbProducts> logger)
+        {
+            _dynamoDbClient = dynamoDbClient;
+            _logger = logger;
+        }
+
+        public async ValueTask Prime()
```

Comment:
Is this used?

Reply:
Oh, no this was my first implementation. It can be removed